rpcs3/rpcs3qt/vfs_dialog_tab.h

```diff
@@ -23,20 +23,24 @@ class vfs_dialog_tab : public QWidget
 	Q_OBJECT
 
 public:
-	explicit vfs_dialog_tab(const vfs_settings_info& info, std::shared_ptr<gui_settings> guiSettings, std::shared_ptr<emu_settings> emuSettings, QWidget* parent = nullptr);
+	explicit vfs_dialog_tab(vfs_settings_info info, std::shared_ptr<gui_settings> guiSettings, std::shared_ptr<emu_settings> emuSettings, QWidget* parent = nullptr);
 
 	void SetSettings();
-	void AddNewDirectory();
 
 	// Reset this tab without saving the settings yet
 	void Reset();
 
 private:
+	void AddNewDirectory();
+	void RemoveDirectory();
+	void SetCurrentRow(int row) { m_currentRow = row; }
```

Comment:
why this private setter?
seems like bloating

Reply:
Right, I made it assuming lambda wouldn't have access to this field otherwise, but this setter is also private. D'oh.

Reply:
Do you reckon it really needs to be changed though?

Reply:
no. just pointing it out.
unless you wanna rebase now

Reply:
if you don't wanna add more i'll merge it afterwards

Reply:
I think I'll pass on that. Let's keep it as is.